### PR TITLE
update notebooks to lcviz 0.4.2

### PR DESCRIPTION
**Pull request by kecnry**
as well as some general cleanup to use public API calls, etc.

### CODE REVIEW

**Review by bmorris3 — APPROVED** [review]
I'm approving with just one edit that you need before it's merged.

In the short cadence notebook, you give a data label to the stitched light curve, but try to edit the plot options on a layer that used to exist when we loaded quarters separately, causing a failure:
```
ValueError: HAT-P-7 Q3 not one of ['HAT-P-7', 'binned HAT-P-7:default'], reverting selection to HAT-P-7
```

I've made suggestions to fix this.

Should we consider putting this repo as a submodule in the lcviz repo?

File: short_cadence_phase_curve_HAT-P-7.ipynb
Comment (kecnry) [reply short_cadence_phase_curve_HAT-P-7.ipynb:163]: hmmm are you on main or 0.4.2?  I am pretty sure I ran this through and the quarter was appended to the input data-label.  (0.4.x does not have https://github.com/spacetelescope/lcviz/pull/120)

**Pull request merged**
